### PR TITLE
Update to Blacklight 7

### PR DESCRIPTION
**Pull request by ebenenglish**
This PR attempts to bring the project up to date with Blacklight 7.*.

Expect a few more commits to come, while Travis build settings are tweaked as needed.

Thanks!

### CODE REVIEW

File: app/assets/images/blacklight/maps.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
+  <image id="Ellipse_1" data-name="Ellipse 1" x="5" y="1" width="14" height="22" xlink:href="data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAWCAYAAADwza0nAAAA/klEQVQ4jZWSMU4CQRSGP8UOGw9gVilMMJEzmG2x1EjhESy9AOEAWJBwAyhI6Gih4gJgZcUNSAg1mIE3yfDeW4Ev+beY//9n3+zOBZYHoA48iTMHRsCvSSZkQAdYAFvRQtYykxbegU1S0NpI5oDcCRYpj8VLYF0Q8rSWDh+OGTQVeV7o0HOMVnKOluP3gzFxDI32x2HWpYnt/2PkxbiwLAFV4FkZDfkAYb0L6IsyCI+KM8ox3ccdvs8otfXcKyektTKnBV6doNabaQkzJxz1Y9IJj04hqmbSir5TGpqUQ9kp3pxSDHwlpeappUjRvd1x9U/xE7g+922BW+DOdYA/IUiSrESa4EIAAAAASUVORK5CYII="/>
```
Comment (mejackreed) [inline app/assets/images/blacklight/maps.svg:2]: We might want to remove the `id` and `data-name`. Is it possible to not have embedded images and use svg shapes instead? Adopters may want to customize the colors downstream.

File: app/assets/stylesheets/blacklight_maps/default.scss [hunk 2]
```diff
@@ -1,6 +1,8 @@
 @import 'leaflet';
-@import 'leaflet.markercluster';
-@import 'leaflet.markercluster.default';
+@import 'MarkerCluster';
+@import 'MarkerCluster.Default';
+//@import 'leaflet.markercluster';
```
Comment (mejackreed) [inline app/assets/stylesheets/blacklight_maps/default.scss:4]: Is this commented out import needed?

File: app/assets/stylesheets/blacklight_maps/default.scss [hunk 3]
```diff
@@ -13,22 +15,22 @@ body.blacklight-catalog-map {
   }
 
 }
-
+// TODO: I think below is no longer needed?
```
Comment (mejackreed) [inline app/assets/stylesheets/blacklight_maps/default.scss:18]: Should we remove?

File: config/locales/blacklight-maps.en.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@ en:
         point_search: 'View items from this location'
         search_ctrl_cue: 'Search for all items within the current map window'
       title: 'Map'
-      leader: 'Click on a marker to search for items from that location.'
+      leader_html: "Haga clic en un marcador para buscar elementos desde esa ubicación, o use el &#128269; botón para buscar todos los elementos dentro de la ventana del mapa actual."
```
Comment (mejackreed) [inline config/locales/blacklight-maps.en.yml:12]: Is this the correct value here for the `en` locale?

File: vendor/assets/javascripts/leaflet.js.erb [hunk 5]
```diff
@@ -0,0 +1,13922 @@
+//= depend_on_asset "marker-icon-2x.png"
```
Comment (mejackreed) [inline vendor/assets/javascripts/leaflet.js.erb:1]: Generally i would prefer an approach that didn't include this in vendor. However, I realize there were some issues that were encountered.

**Comment by ebenenglish** [comment]
@mejackreed Thanks for the review!

All comments should be fixed via https://github.com/projectblacklight/blacklight-maps/pull/91/commits/5f4cce2b2733ed1c617b65ea4a9e90afabfa731e, with the exception of the inclusion of `leaflet.js.erb` in `vendor/assets/javascripts`.

I'm not in love with that solution either, I was basically copying the pattern currently used by geoblacklight. Hopefully we can come up a better shared solution down the road?

**Pull request merged**